SportBot.py

```diff
@@ -0,0 +1,107 @@
+import mysql.connector
+import math
+
+mydb = mysql.connector.connect(
+    host="127.0.0.1",
+    user="root",
+    password="sesso6969",
+    database="sportBot"
+)
+
+conn = mydb.cursor()
+conn.execute("CREATE TABLE IF NOT EXISTS workouts (id BIGINT AUTO_INCREMENT PRIMARY KEY, user_id BIGINT, time TEXT, distance FLOAT)")
+conn.execute(
+    "CREATE TABLE IF NOT EXISTS goals (id BIGINT AUTO_INCREMENT PRIMARY KEY,user_id BIGINT, distance FLOAT)")
```

Comment:
Non è molto elegante mettere questo tipo di operazioni al livello base, dato che così sarebbero eseguite anche ogni volta che qualcuno importa questo modulo. Conviene racchiuderle in una funzione, anche del bot, e chiamarla esplicitamente, se lo si ritiene necessario.